posthog/models/remote_config.py

```diff
@@ -91,6 +92,21 @@ def sanitize_config_for_public_cdn(config: dict, request: Optional[HttpRequest]
     return config
 
 
+def _should_have_custom_rrweb_script(team_id: int) -> bool:
+    if settings.SESSION_REPLAY_RRWEB_SCRIPT is None:
+        return False
+
+    force_enable_list = settings.SESSION_REPLAY_RRWEB_SCRIPT_FORCE_ENABLE_TEAMS
+    if force_enable_list and team_id in force_enable_list:
+        return True
```

Comment:
we can set specific teams to always get the named script
primarily this will be the posthog team
but might be necessary in response to support